development/custom-image/create-custom-image.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+set -o errexit
+
+readonly CURRENT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+readonly ROOT_DIR=${CURRENT_DIR}/../../
+# shellcheck disable=SC1090
+source "${ROOT_DIR}/prow/scripts/library.sh"
+
+cleanup() {   
+    shout "Removing instance kyma-deps-image-vm-${RANDOM_ID}"
+    gcloud compute instances delete --quiet --zone "${ZONE}" "kyma-deps-image-vm-${RANDOM_ID}"
+}
+
+
+RANDOM_ID=$(openssl rand -hex 4)
+DATE=$(date +v%Y%m%d)
+ZONE=$(openssl rand -hex 4)
+
+ZONE_LIMIT=${ZONE_LIMIT:-5}
+EU_ZONES=$(gcloud compute zones list --filter="name~europe" --limit="${ZONE_LIMIT}" | tail -n +2 | awk '{print $1}')
+for ZONE in ${EU_ZONES}; do
+    shout "Attempting to create a new instance named kyma-deps-image-vm-${RANDOM_ID} in zone ${ZONE}"
+    gcloud compute instances create "kyma-deps-image-vm-${RANDOM_ID}" \
+        --metadata enable-oslogin=TRUE \
+        --machine-type n1-standard-4 \
+        --image debian-9-stretch-v20181011 \
```

Comment:
We are tagging a specific version of the OS as our base system. With this we will have the same problem that before, that if we run this script in a month or a few, this image will be no longer available. We should tag to the latest debian from the 9 family, or better yet set this as a configurable env.

Reply:
Updated the tag to become  --image-family debian-9